opengrok-tools/src/test/python/test_mirror.py

```diff
@@ -109,3 +114,59 @@ def test_valid_config():
     assert check_configuration({PROJECTS_PROPERTY:
                                 {"foo": {PROXY_PROPERTY: True}},
                                 PROXY_PROPERTY: "proxy"})
+
+
+@pytest.mark.skipif(not os.name.startswith("posix"), reason="requires posix")
```

Comment:
any reason why this should work only on posix?

Reply:
should work on Windows too, let me try.

Reply:
hangs in the newly added test for some reason, will keep posix for now.

Reply:
ok